api/services/elastigroup/azure/spotVms/schemas/elastigroupVm-base-scheduling-task.yaml

```diff
@@ -0,0 +1,16 @@
+type: object
+required:
+  - isEnabled
+  - cronExpression
+properties:
+  isEnabled:
+    type: string
+    description: |
+      A list of scheduling tasks.
+  cronExpression:
+    type: string
+    description: |
+      A valid cron expression.
+      The cron is running in UTC time zone and is in Unix cron format of Cron Expression Validator Script.
+      Only one of 'frequency' or 'cronExpression' should be used at a time.
```

Comment:
Can you define enum here for the allowed values?

Reply:
This isn't an enumeration; this is 2 properties that at least one of them must be set.